### ISSUE TITLE
Does this plugin sends push notification while app is running in background?

### ISSUE THREAD

**Issue by vithanidarpan** [phonegap/phonegap-plugin-push#1041]
I have used this plugin and it works fine when app is open.
But when app is in background, $cordovaPushV5:notificationReceived method is not working.
I'm not getting notification when app is running in background.

Or is there any option that I need to set for receiving notification in background?

**Comment by macdonst** [comment]
@vithanidarpan yes, yes there is. How to get your notification received handler called when the app is in the background is detailed here:

[Background Events](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#push-message-arrives-with-app-in-background)
- [Android](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#background-notifications)
- [iOS](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#background-notifications-1)
- [Windows](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#background-notifications-2)

**Comment by vithanidarpan** [comment]
Thanks @macdonst 
I used reference you provided. So, now it works in android device. But it is not working in ios. When I checked log in xcode, I got notification object but $cordovaPushV5:notificationReceived method is not working.
When application is in foreground nothing happens and when app is in background I'm getting notification but when I click on that notification nothing happens.
I have written cordova.plugins.notification.local.on("click", function(notification, state) {}) method to handle notification click event and it works in android.
